plbenchmark/utils.py

```diff
@@ -237,35 +234,35 @@ def convert_error(
     # define default units
     if out_unit is None:
         if final_type == "dg":
-            out_unit = unit_registry("kilocalories / mole")
+            out_unit = unit("kilocalories / mole")
         elif final_type == "ki":
-            out_unit = unit_registry("nanomolar")
+            out_unit = unit("nanomolar")
         elif final_type == "ic50":
-            out_unit = unit_registry("nanomolar")
+            out_unit = unit("nanomolar")
         elif final_type == "pic50":
-            out_unit = unit_registry("")
+            out_unit = unit("")
 
     if original_type == "dg":
         if final_type == "dg":
             return error_value.to(out_unit)
         elif final_type == "ki":
             # e_ki^2 = (del K/del dG)^2 * e_dG^2
             # e_ki   = 1/RT * exp(-dG/RT) * e_dG
-            k_bt = boltzmann_constant * temperature * unit_registry.kelvin
+            k_bt = unit.molar_gas_constant * temperature * unit.kelvin
             error = (
-                1.0 / k_bt * np.exp(-value / k_bt) * error_value * unit_registry.molar
+                1.0 / k_bt * np.exp(-value / k_bt) * error_value * unit.molar
             )
             return error.to(out_unit)
         elif final_type == "ic50":
-            k_bt = boltzmann_constant * temperature * unit_registry.kelvin
+            k_bt = unit.molar_gas_constant * temperature * unit.kelvin
```

Comment:
@IAlibay are we certain we want `R` here instead of `k`? Trying to check that the downstream uses of `k_bt` don't assume that it's a per mol quantity, but not clear to me on a quick pass.

Reply:
So the old version of the code was using boltzmann_constant = 8.314462618 J / mole / K, which is R.

This makes sense based on the comments in the code and what I remember from my chemistry lectures but a second eye wouldn't hurt either.

Reply:
All good, yes you're exactly right. Just chased it down as well. Thanks for fixing this!